playground/frontend/lib/config.g.dart

```diff
@@ -0,0 +1,25 @@
+/*
+ * Licensed to the Apache Software Foundation (ASF) under one
+ * or more contributor license agreements.  See the NOTICE file
+ * distributed with this work for additional information
+ * regarding copyright ownership.  The ASF licenses this file
+ * to you under the Apache License, Version 2.0 (the
+ * "License"); you may not use this file except in compliance
+ * with the License.  You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+// This file was auto-generated and used to contain backend URLs.
+// They are now determined automatically
+// as of https://github.com/apache/beam/pull/25610
+// TODO(alexeyinkin): When it is merged, find a better place for the remaining
+//  constant and delete this file.
+
+const String kAnalyticsUA = 'G-BXFP2FNCKC';
```

Comment:
Rename to `config.dart` or similar as it no longer is a generated file

Reply:
This is to prevent merging conflicts because there are other PRs to change this file. When analytics lands, I will.